var/spack/repos/builtin/packages/crunch/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Crunch(CMakePackage):
+    """Advanced DXTc texture compression and transcoding library."""
+
+    homepage = "https://github.com/BinomialLLC/crunch"
+    # Fork with CMake build system
+    git = "https://github.com/rouault/crunch.git"
```

Comment:
I am a bit concerned adding a 0 star repository as git. Is this repo used by people? Can't we try to push for https://github.com/BinomialLLC/crunch/pull/33 being merged?

Reply:
Completely understand your concern. Let me add some background.

I have no interest in crunch, I only added it as a GDAL dependency. The [GDAL docs](https://gdal.org/build_hints.html#crnlib) explicitly point to this branch of this fork (presumably due to its ease of installation). The [original fork](https://github.com/BinomialLLC/crunch/graphs/contributors) seems poorly maintained at best. Most PRs and issues have no comments.

This fork was created by @rouault, who also maintains GDAL, so I think I actually trust it more than the original fork, especially for use with GDAL.

@BinomialLLC is there any chance of https://github.com/BinomialLLC/crunch/pull/33 being merged so we can use the original repo instead? Also, would you have any interest in maintaining this build recipe in Spack?

Reply:
Given that the upstream poackage seems not to be maintained, how many patches are we talking about? If they're just a few, can we pin a commit from upstream and cherry pick changes from the other repo as patches?

Reply:
We are talking about 5 patches: https://github.com/rouault/crunch/commits/build_fixes

We could use upstream and add these 5 commits as patches, but I'm not sure how much of an advantage that would be.

Reply:
@alalazo how do you want to progress here?